.github/workflows/build.yml

```diff
@@ -0,0 +1,52 @@
+on:
+  push:
+    branches: [master,github-actions-test]
+  pull_request:
+    branches: [master]
+
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.7]
```

Comment:
Yes to add more versions: 3.6, 3.7, 3.8 (3.9)? With more recent versions of python, we might lose dependencies: for example, TensorFlow is not yet available in python 3.9: https://www.tensorflow.org/install/pip